Formula/nvc.rb

```diff
@@ -31,23 +31,15 @@ class Nvc < Formula
 
   resource "homebrew-test" do
     url "https://github.com/suoto/vim-hdl-examples.git",
+        branch:   "master",
```

Comment:
This isn't needed, since we specify `revision`.

Reply:
Interesting, `audit` complained to me about the lack of `branch` on Linux.